library_generation/README.md

```diff
@@ -105,29 +105,31 @@ The library level parameters define how to generate a (multi-versions) GAPIC
+| product_docs          |   Yes    |                                                                                                                 |
+| library_type          |    No    | `GAPIC_AUTO` if not specified                                                                                   |
+| release_level         |    No    | `preview` if not specified                                                                                      |
+| api_id                |    No    | `{api_shortname}.googleapis.com` if not specified                                                               |
+| api_reference         |    No    |                                                                                                                 |
+| codeowner_team        |    No    |                                                                                                                 |
+| client_documentation  |    No    |                                                                                                                 |
+| distribution_name     |    No    | `{group_id}:google-{cloud_prefix}{library_name}` if not specified                                               |
+| excluded_poms         |    No    |                                                                                                                 |
+| excluded_dependencies |    No    |                                                                                                                 |
+| googleapis_commitish  |    No    | use repository level `googleapis_commitish` if not specified.                                                   |
+| group_id              |    No    | `com.google.cloud` if not specified                                                                             |
+| issue_tracker         |    No    |                                                                                                                 |
+| library_name          |    No    | `api_shortname` is not specified. This value should be unique among all libraries.                              |
+| rest_documentation    |    No    |                                                                                                                 |
+| rpc_documentation     |    No    |                                                                                                                 |
+| cloud_api             |    No    | `true` if not specified                                                                                         |
+| requires-billing      |    No    | `true` if not specified                                                                                         |
+| transport             |    No    | must be one of `grpc`, `rest`, `http` (alias) or `both`. This will override the value obtained from BUILD.bazel |
```

Comment:
I think we use `transport` from BUILD.bazel to determine if we should generate a grpc module or not, can you please confirm that? If yes, we should update the description to something like `This value would only be used for generating .repo-metadata.json and relevant sections in README`.

Reply:
Done